worldedit-core/src/main/java/com/fastasyncworldedit/core/command/brush/TargetMode.java

```diff
@@ -1,4 +1,4 @@
-package com.fastasyncworldedit.core.object.brush;
+package com.fastasyncworldedit.core.command.brush;
```

Comment:
Feels wrong here as just an enum... Lost in the bushes

Reply:
Its only use is in worldedit.command.tool.BrushTool. Maybe move it (back) to the named package referenced via `package-info.java` ?

Reply:
command.enum maybe? Random enums in a package feels wrong

Reply:
Eh, `enum` is somewhat of reserved so rather not do that. We have enums all around the place in existing packages.